src/pint/models/glitch.py

```diff
@@ -159,8 +158,8 @@ def glitch_phase(self, toas, delay):
         glepnames = [x for x in self.params if x.startswith("GLEP_")]
         for glepnm in glepnames:
             glep = getattr(self, glepnm)
-            eph = glep.value
             idx = glep.index
+            eph = glep.quantity.to_value("mjd", "long")
```

Comment:
Sorry I realize this is new feature of astropy 4.0. But, glep.value should do the same thing.

Reply:
It doesn't do the same thing -- value returns a 64-bit float.  Doing it the above way returns a longdouble.  And I think this is behavior that probably want in other places in the code.  But that would require astropy 4, I guess.  It is a nice new feature....

Reply:
If value returns 64-bit float, there is a bug in our code. Since the MJD parameter .value use our util function time_to_longdouble. But if astropy 4.0 provides this function. we should use the astropy version.

Reply:
Then I think we have a bug (and I actually saw that in the code and worried about it).  Look at the following:
```
In [52]: tm = t.Time('51544.000000000000002', format='mjd')                     

In [53]: tm.to_value('mjd', subfmt='decimal')                                   
Out[53]: Decimal('51544.00000000000000199840144432528')

In [54]: tm.to_value('mjd', subfmt='long')                                      
Out[54]: 51544.000000000000004

In [55]: type(tm.to_value('mjd', subfmt='long'))                                
Out[55]: numpy.float128

In [56]: tm.mjd, type(tm.mjd), (tm.mjd-51544)==0                                
Out[56]: (51544.0, numpy.float64, True)

In [57]: tm.value, type(tm.value), (tm.value-51544)==0                          
Out[57]: (51544.0, numpy.float64, True)

In [58]: import astropy                                                         

In [59]: astropy.__version__                                                    
Out[59]: '4.0'
```

Reply:
I see you are talking about the astropy time object value. If it is a pint.model.parameter MJD parameter type. It will do the same. 
```
from pint.models.parameter import perfixParameter
In [7]: x = prefixParameter(name="GLEP_1", units="MJD", value='51544.00000000000
   ...: 0002', parameter_type="MJD",  time_scale="tdb")                         
In [8]: x.value                                                                 
Out[8]: 51544.000000000000004

In [9]: type(x.value)                                                                       
Out[9]: numpy.float128
```

Reply:
OK, that's good to know.